bazel/emscripten_toolchain/emar.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source emscripten_toolchain/env.sh
+
+exec python3 $EMSCRIPTEN/emar.py "$@"
```

Comment:
I wonder if we can (one day) find a way to drop the extra wrapper scripts and just use the normal `$EMSCRIPTEN/emar` entry points that are part of emscripten.